pkg/ioutils/lazy_reader_disk.go

```diff
@@ -2,7 +2,6 @@ package ioutils
 
 import (
 	"io"
-	"io/ioutil"
```

Comment:
`io/ioutil` is deprecated, so might as well remove it here while I'm touching this file